### DISCUSSION TITLE
Vulnerabilities in docker images

### DISCUSSION THREAD

**Discussion by josteint** [open-metadata/OpenMetadata#17409] [Q&A] [answered]
I installed OpenMetaData in Docker Desktop 1.4.7 on Windows. I looked into the mysql image and it has 177 vulnerabilities. The elasticsearch image has 80 vulnerabilities. Am I doing something wrong?

**Comment by Prajwal214** [comment]
For Production Deployment using Docker, we recommend bringing your own Databases and ElasticSearch Engine and not rely on quickstart packages.

https://docs.open-metadata.org/latest/deployment/docker#configure-openmetadata-to-use-external-database-and-search-engine

**Comment by harshach** [comment, marked as answer]
@josteint we do not control the mysql or elasticsearch images, those communities are responsible for publishing those images. If there are any vulnerabilities in opemetadata-server we will make sure with every release there are zero vulnerabilities 
As @Prajwal214 describe for production deployment, make sure you are depending on a external saas or using a docker image that will work for you from the respective community
